crates/containerd-shim-wasm/src/sandbox/shim.rs

```diff
@@ -1300,15 +1303,7 @@ where
             shim::Error::InvalidArgument(format!("error loading runtime spec: {}", err))
         })?;
 
-        let default = HashMap::new() as HashMap<String, String>;
-        let annotations = spec.annotations().as_ref().unwrap_or(&default);
-
-        let id = opts.id.clone();
-
-        let grouping = annotations
```

Comment:
Why is this deleted?

Reply:
@hydai I saw this grouping logic was changed from Lukic's reference code for fix networking issue. Could you help answer it?

Reply:
This was a mistake, sorry, we didn't need anything k8s related so I removed it during testing, this should be reverted

Reply:
Copy that. I will help revert this.

Reply:
Revert it done.